Dockerfile

```
@@ -1,14 +1,15 @@
+FROM quay.io/yshnaidm/livecd-iso:fedora-coreos-livecd
```

Why not replace this with:
```suggestion
FROM alpine:latest
RUN mkdir /root/image
COPY ./fedora-coreos-31.20200319.dev.1-live.x86_64.iso /root/image/livecd.iso```